src/generator.ts

```diff
@@ -1,13 +1,11 @@
 import { createFile, Feature, Prototyper, Widget } from 'concordialang-ui-core'
-import * as fs from 'fs'
 import { promisify } from 'util'
 const pretty = require('pretty')
 
 import WidgetFactory from './widgets/widget-factory'
 
 export default class Generator implements Prototyper {
-
-  constructor(private _fs: any = fs) {
+  constructor(private _outputDir: string) {
```

Comment:
Receiving the `fs` is important for testing purposes.
The `fs` object should be used in the file creation.

Reply:
The `fs` is not necessary anymore. The `ui-core` provides a function to create the file.